### PR TITLE
Add ability to manage scrapes

### PR DESCRIPTION
**Pull request by cguess**
This adds the ability to view, resubmit and delete scrapes that have not
been completed yet.

To test:
1. `rails db:migrate`
1. `rails test`

### CODE REVIEW

File: config/tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+module.exports = {
+  content: [
+    './app/helpers/**/*.rb',
+    './app/javascript/**/*.js',
+    './app/views/**/*.{erb,haml,html,slim}'
+  ],
+  theme: {
+    extend: {
+      fontFamily: {
+        sans: ['Inter var', ...defaultTheme.fontFamily.sans],
```
Comment (reefdog) [inline config/tailwind.config.js:12]: Just to sanity-check `Inter var` is the valid name of the font?

File: app/views/jobs_status/index.html.erb [hunk 2]
```diff
@@ -18,3 +19,36 @@
...
+    <tbody id="jobs-table-body">
+      <% @scrapes.each do |scrape| %>
+        <tr class="border text-gray-600 text-center">
+          <td><%= scrape.id %></td>
+          <td><%= scrape.scrape_type %></td>
+          <td><%= scrape.created_at %></td>
+          <td><%= link_to scrape.url, scrape.url %></td>
+          <td><%= link_to "Delete", job_status_delete_path(scrape.id), data: {
+                    turbo_method: :delete,
+                    turbo_confirm: "Are you sure?"
```
Comment (reefdog) [inline app/views/jobs_status/index.html.erb:44]: UX request: can you customize this confirmation so it's more clear to the user which action they're confirming? Suggested: "Are you sure you want to delete this job?"

File: app/views/jobs_status/index.html.erb [hunk 3]
```diff
@@ -18,3 +19,36 @@
...
+                  } %></td>
+          <td><%= link_to "Resubmit", job_status_resubmit_path(scrape.id), data: {
+                    turbo_method: :post,
+                    turbo_confirm: "Are you sure?"
```
Comment (reefdog) [inline app/views/jobs_status/index.html.erb:48]: UX request: can you customize this confirmation so it's more clear to the user which action they're confirming? Suggested: "Are you sure you want to resubmit this job?"

**Review by reefdog — CHANGES_REQUESTED** [review]
Very small UX request around confirmation language.

Also, when I run this I get a bunch of failing tests, but I suspect this is due to an incomplete setup. I'll resolve that with you separately (and update the README for #156 further as needed), and then check this again once my setup actually works.

File: config/tailwind.config.js (hunk 1 above)
Comment (reefdog) [reply config/tailwind.config.js:12]: Answering my own question here: [yup, `Inter var` is correct](https://tailwindui.com/documentation#optional-add-the-inter-font-family).

Comment (reefdog) [reply config/tailwind.config.js:12]: Answering my own question here: [yup, `Inter var` is correct](https://tailwindui.com/documentation#optional-add-the-inter-font-family).

**Comment by cguess** [comment]
Yes. I’ll get to it in the next hour

-Christopher Guess
***@***.***
US/WhatsApp/Signal: +1 262.893.1037
PGP: AAE7 5171 0D81 B45B – https://keybase.io/cguess

**Comment by cguess** [comment]
All cleaned up. @reefdog please take a look

**Comment by reefdog** [comment]
@cguess Tests are currently running, but I just fired up the app and was greeted with a `NameError in Archive#index`, I think because of the same `jobs_status_path` issues you were fixing in 2db1b8da1ed4944a719b41cdb5682f71699e9be7. Might need a few more snipes:

image

**Review by reefdog — CHANGES_REQUESTED** [review]
@cguess It's a clean test run! 🎉 The only issue is the error on `jobs_status_path` that I noted above, that causes rendering the header to throw an error. After that I think this is good to go.

…although it strikes me that I don't actually know how to test the _functionality_ of this in the browser.

**Review by reefdog — APPROVED** [review]

**Pull request merged**